beluga_example/rviz/ndt_rviz.ros2.rviz

```diff
@@ -0,0 +1,289 @@
+Panels:
```

Comment:
@serraramiro1 nit^N!: maybe it's about time we rename rviz config files to match their use: `ndt_amcl.ros2.rviz` here, `amcl.ros*.rviz` elsewhere.